MathLib/LinAlg/Lis/LisOption.h

```diff
@@ -40,12 +41,13 @@ namespace MathLib
  */
 struct LisOption
 {
-	LisOption(BaseLib::ConfigTree const* const options)
+	LisOption(BaseLib::ConfigTreeNew const* const options)
 	{
 		if (options)
 		{
-			_option_string += " " + options->get<std::string>("lis", "");
-			boost::algorithm::trim_right(_option_string);
+			ignoreOtherLinearSolvers(*options, "lis");
+			if (auto s = options->getConfParamOptional<std::string>("lis"))
+				_option_string += " " + *s;
```

Comment:
I wondered, why it was right-trimmed and didn't think about empty config params.
Maybe a line of comment would have helped me to understand the motivation of the trim.
Anyway, now it's fixed.